docs/guides/elasticsearch/quickstart/overview/elasticsearch/index.md

```diff
@@ -141,6 +141,8 @@ The KubeDB operator implements an Elasticsearch CRD to define the specification
 
 The Elasticsearch instance used for this tutorial:
 
+If your `KubeDB version` is greater than `v2024.6.4` use `v1` apiVersion.
+
```

Comment:
"If your `KubeDB version` is less or equal to `v2024.6.4`, You have to use `v1alpha2` apiVersion."
In all places.